libs/api/domains/cms/src/lib/generated/contentfulTypes.d.ts

```diff
@@ -301,6 +301,42 @@ export interface IArticleSubgroup extends Entry<IArticleSubgroupFields> {
   }
 }
 
+export interface IAuctionFields {
+  /** Title */
+  title: string
+
+  /** Date */
+  date?: string | undefined
```

Comment:
I think adding `undefined` is redundant since this field is already optional

Reply:
This file is autogenerated using `yarn nx run api:contentful-types`, we could change that script but maybe we should just let it be? :)